compiler/testData/codegen/box/coroutines/debug/elvisLineNumber.kt

```diff
@@ -28,14 +27,14 @@ fun box(): String {
         withLineBreak()
         return "FAIL 0"
     } catch (e: RuntimeException) {
-        if (e.stackTrace[0].lineNumber != 19) return "FAIL 1 ${e.stackTrace[0].lineNumber}"
```

Comment:
I would prefer to avoid unecessery test data changes: so it's best to add new line after // WITH_COROUTINES

Reply:
Thanks Mike! Done.